force-app/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -19,632 +19,4 @@
         <visibility>DefaultOn</visibility>
     </tabVisibilities>
     <userLicense>Salesforce</userLicense>
-    <userPermissions>
```

Comment:
@andreysaf The `<userPermissions>` tags are not required to make the demo run - removing them as some of these are deprecated and cause deployment issues on newer Salesforce instances